### PR TITLE
Download the body fat dataset

### PR DESCRIPTION
**Pull request by shrit**
We need to add bodyfat dataset into examples in dataset
possible link:
http://staff.pubhealth.ku.dk/~tag/Teaching/share/data/Bodyfat.html

Signed-off-by: Omar Shrit <omar@shrit.me>

### CODE REVIEW

**Comment by kartikdutt18** [comment]
Hey @shrit, Thanks for this. The dataset can be found [here](https://drive.google.com/drive/folders/1GL0Somp6Exupb-vdI0VwDiZUKIaHgPfo?usp=sharing).

**Comment by shrit** [comment]
@kartikdutt18 Perfect I will update extensions.

**Comment by shrit** [comment]
@kartikdutt18 you are welcome

File: tools/download_data_set.py [hunk 1]
```diff
@@ -120,6 +120,11 @@ def stock_exchange_dataset():
   stock = requests.get("https://www.mlpack.org/datasets/examples/Google2016-2019.csv")
   progress_bar("stock.csv", stock)
 
+def body_fat_dataset():
+  print("Download the body fat datasets")
+  bodyFat = requests.get("https://www.mlpack.org/datasets/examples/bodyfat.tsv")
```
Comment (birm) [inline tools/download_data_set.py:125]: I'm getting a 404 at this url. Are you intending to add it?

Comment (shrit) [reply tools/download_data_set.py:125]: Yes it is intended, we need to add the dataset to the site.

Comment (shrit) [reply tools/download_data_set.py:125]: @rcurtin when you have the time just please add the dataset into example folder on mlpack website

Comment (rcurtin) [reply tools/download_data_set.py:125]: Done, sorry for the delay!  http://www.mlpack.org/datasets/examples/  -- let me know if there are any issues. :+1:

**Pull request merged**